shared/schemas/workflow-status-schema.json

```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowStatus",
+    "description": "Status of workflow execution",
+    "type": "object",
+    "properties": {
+        "status": {
+            "description": "The current status of a workflow execution",
+            "type": "string",
+            "enum": [
+                "running",
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "An array of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "description": "A next workflow to run.",
+                "type": "object",
+                "properties": {
+                    "title": {
+                        "description": "Human readable title describing the workflow, e.g. its name.",
+                        "type": "string"
+                    },
+                    "workflowId": {
+                        "description": "Identification of the workflow",
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "title",
+                    "workflowId"
+                ]
+            }
+        },
+        "options": {
+            "description": "Additional structured output of workflow processing.",
+            "type": "array",
+            "items": {
+                "description": "An item is either key:value or key:link pair.",
+                "type": "object",
+                "properties": {
+                    "key": {
+                        "description": "Unique identifier of the option. Preferably human-readable.",
+                        "type": "string"
+                    },
+                    "value": {
+                        "description": "Free form value of the option.",
+                        "type": "string"
+                    },
+                    "link": {
```

Comment:
Would it make sense to make JSON/nested object as explicit link the link here? Is there a use-case for it?

Reply:
as far as I remember it was added to provide references to third party systems

Reply:
@mareklibra I'm not sure I understand the use-case.
Do you mean to have a value with a content of a JSON?

Links are pretty common in this context (for Jira, MTA report,...). In MTA v7, the link is for downloading a zip file.
This structure doesn't specify the content of the value, it can be okay, but I'd like to verify when aligning data in the `workflow results` panel, isn't there  a need to figure out if a field is a text, number, date... for better presenting it?
I'm happy with the simplicity of this, but would like to verify it is fine from the frontend side.

Reply:
The point is to provide a way to share complex result. The UI would not parse it, just show the json, ready to copy&paste. It can be a snippet of a config file or other data object. Since this can be simulated by a `text` field, let's skip this idea for now. Can be easily added later, if ever needed.

Reply:
An idea for the future: can this be a set of inputs for a next workflow?